docs/manuals/core/concepts/task.md

```diff
@@ -29,8 +28,8 @@ the task configuration `TaskConfig^` that must be provided when instantiating a
 
 
 !!! Important
-    Since no task computes them, _sales history_, _current month_, and _capacity_ are considered as **input**
-    data nodes. <br>
+    The data nodes _sales history_, _current month_, and _capacity_ are considered as **input**
+    data nodes since no task computes them.<br>
```

Comment:
Please get used to using `<br/>` instead of `<br>`. That may be very handy should we need to parse that.